src/prebid.js

```diff
@@ -101,17 +101,20 @@ function checkDefinedPlacement(id) {
 }
 
 function getWinningBidTargeting() {
-  const presets = (function getPresetTargeting() {
-    return window.googletag.pubads().getSlots().map(slot => {
-      return {
-        [slot.getAdUnitPath()]: slot.getTargetingKeys().map(key => {
-          return { [key]: slot.getTargeting(key) };
-        })
-      };
-    });
-  })();
+  let presets;
+  if (utils.isGptPubadsDefined()) {
```

Comment:
You can import `isGptPubadsDefined` as a named export from `utils` as we do with `flatten`, `uniques` et. al. at the top of this module.